webpack/assets/javascripts/react_app/routes/Models/ModelsPage/ModelsPage.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Button } from 'patternfly-react';
+
+import { getParams } from '../../../common/urlHelpers';
+import PageLayout from '../../../pages/common/PageLayout/PageLayout';
+import ModelsTable from '../../../components/ModelsTable';
+import { withRenderHandler } from '../../../common/HOC';
+
+import searchProps from '../consts';
+
+const ModelsPage = props => (
+  <PageLayout
+    header={__('Hardware Models')}
+    searchable={true}
+    searchProps={searchProps}
+  >
+    <ModelsTable />
+  </PageLayout>
+);
+
+export default withRenderHandler({
+  Component: ModelsPage
```

Comment:
Looks so clean now, awesome!

one issue - `withRenderHandler` should wrap `ModelsTable` and not the whole `PageLayout`,
so when an error/empty/loading occurs, the user can still see the PageLayout ( header, search, breadcrumbs etc)